frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -186,11 +190,11 @@ Browser: ${window.navigator.userAgent}
 `);
   return _.isEmpty(prerelease)
     ? {
-        label: 'Open Support Case with Red Hat',
+        label: t('public~Open Support Case with Red Hat'),
```

Comment:
You need to do something like `i18next.t('public~Open Support Case with Red Hat')` here instead.